src/test/java/tests/inputTest.java

```diff
@@ -0,0 +1,31 @@
+package tests;
+
+import org.testng.annotations.Test;
+import static org.openqa.selenium.Keys.ARROW_DOWN;
+import static org.openqa.selenium.Keys.ARROW_UP;
+import static org.testng.Assert.assertEquals;
+
+public class inputTest extends BaseTest {
```

Comment:
Name of class should start from Capital letter

Reply:
done